### PR TITLE
feat(Window): add "active" state to window action

### PR DESCRIPTION
**Pull request by cnairi**
## DESCRIPTION

Add an active state for Window header button.

## HOW TO TEST

http://localhost:3020/components/window#header > Active button

click on the "setting" icon

## SCREENSHOTS / SCREEN RECORDINGS

Active
Capture d’écran 2026-06-10 à 14 59 14

Not active
Capture d’écran 2026-06-10 à 14 59 22

## COMPATIBILITY

- [x] Tested on Safari (desktop)
- [x] Tested on Chrome (desktop)
- [x] Tested on Firefox (desktop)
- [x] Tested on mobile device sizes
- [x] Tested on tablet device sizes
- [ ] Tested on IOS Safari (either device or simulator)

## QA

- [x] Thoroughly tested in local environment
- [x] Added tests for all new features
- [x] Added tests that considered edge cases

### CODE REVIEW

File: lib/src/components/Window/components/Header.tsx [hunk 1]
```diff
@@ -24,12 +24,14 @@ const Button = ({
...
   className,
   icon,
+  isActive = false,
   onClick,
   ...rest
 }: PropsWithAs<As, ActionButtonProps>) => {
   return (
     <button
       aria-label={ariaLabel}
+      aria-pressed={isActive}
```
Comment (jonathan-herbault) [inline lib/src/components/Window/components/Header.tsx:34]: Hey Cam, not a big deal but i think you need to adjust this. 
To assistive technology, aria-pressed="false" is not neutral — it explicitly announces the control as a toggle button in the "off" state.

i'll suggest to remove the default `false` value. Keeping `undefined` to omit the props automatically

Comment (cnairi) [reply lib/src/components/Window/components/Header.tsx:34]: thanks for the feedback, I'll do it 🙏

**Review by jonathan-herbault — APPROVED** [review]
